src/locale/locales/de/messages.po

```diff
@@ -556,19 +522,19 @@ msgstr "Konto-Löschung abbrechen"
 
 #: src/view/com/modals/ChangeHandle.tsx:149
 msgid "Cancel change handle"
-msgstr ""
+msgstr "Ändern abbrechen Benutzerhandle"
```

Comment:
This is a mistake. Reading the translations of other similar actions, probably one translation could be

```suggestion
msgstr "Benutzerhandle ändern abbrechen"
```

Reply:
Thanks 🙏